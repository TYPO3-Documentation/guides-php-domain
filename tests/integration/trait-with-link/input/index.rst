=========
PHP Trait
=========

..  php:namespace:: TYPO3\CMS\Core\Context

..  php:trait:: ContextAwareTrait

    ..  php:const:: FLAG_1
        :public:

    ..  php:method:: setContext(Context $context)

    ..  php:method:: getContext()

See also :php:trait:`TYPO3\CMS\Core\Context\ContextAwareTrait`.
