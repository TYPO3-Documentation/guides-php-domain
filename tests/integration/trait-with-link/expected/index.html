<!-- content start -->
    <div class="section" id="php-trait">
            <h1>PHP Trait</h1>

            <dl class="php trait">
    <dt class="sig sig-object php"
        id="typo3-cms-core-context-contextawaretrait">
        <em class="property"><span class="pre">trait</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\Context\</span></span>
<span class="sig-name descname"><span class="pre">ContextAwareTrait</span></span>
<a class="headerlink" href="#typo3-cms-core-context-contextawaretrait" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
        <dl class="php const">
    <dt class="sig sig-object php" id="typo3-cms-core-context-contextawaretrait-flag-1">
        <span class="sig-name modifier"><span class="pre">public</span></span>
 <em class="property"><span class="pre">const</span></em>
        <span class="sig-name descname"><span class="pre">FLAG_1</span></span>

        <a class="headerlink" href="#typo3-cms-core-context-contextawaretrait-flag-1" title="Permalink to this definition">¶</a>
    </dt>
    <dd></dd>
</dl>
<dl class="php method">
    <dt class="sig sig-object php" id="typo3-cms-core-context-contextawaretrait-setcontext">
<span class="sig-name descname"><span class="pre">setContext</span></span>
<span class="sig-paren">(</span>
<em class="sig-param"><span class="pre">Context $context</span></em><span class="sig-paren">)</span>
<a class="headerlink" href="#typo3-cms-core-context-contextawaretrait-setcontext" title="Permalink to this definition">¶</a>
    </dt>
    <dd>

        </dd>
</dl>
<dl class="php method">
    <dt class="sig sig-object php" id="typo3-cms-core-context-contextawaretrait-getcontext">
<span class="sig-name descname"><span class="pre">getContext</span></span>
<span class="sig-paren">(</span>
<span class="sig-paren">)</span>
<a class="headerlink" href="#typo3-cms-core-context-contextawaretrait-getcontext" title="Permalink to this definition">¶</a>
    </dt>
    <dd>

        </dd>
</dl>

    </dd>
</dl>

            <p>See also <a href="/index.html#typo3-cms-core-context-contextawaretrait">\TYPO3\CMS\Core\Context\ContextAwareTrait</a>.</p>
    </div>

<!-- content end -->
